presto-main/src/main/java/io/prestosql/operator/PageTransportErrorException.java

```diff
@@ -13,20 +13,21 @@
  */
 package io.prestosql.operator;
 
-import io.prestosql.spi.PrestoException;
+import io.prestosql.spi.HostAddress;
+import io.prestosql.spi.PrestoTransportException;
 
 import static io.prestosql.spi.StandardErrorCode.PAGE_TRANSPORT_ERROR;
 
 public class PageTransportErrorException
-        extends PrestoException
+        extends PrestoTransportException
```

Comment:
the change here seems not related to `Add remote Host info to PageTransportErrorException`

is it intentional?
make this into a separate commit

also cc @electrum

Reply:
This is needed and related since `PrestoTransportException` is the base class that contains the remote host, which is extracted in `Failures` to include as part of the error info. Given that `PageTransportTimeoutException` already extends it (and is the only subclass), I'm guessing we intended to do it for `PageTransportErrorException` originally.